drivers/scheduler/k8s/k8s.go

```diff
@@ -4100,6 +4118,40 @@ func (k *K8s) ValidateAutopilotRuleObjects() error {
 	return nil
 }
 
+// GetIOBandwidth takes in the pod name and namespace and returns the IOPs speed
+func (k *K8s) GetIOBandwidth(podName string, namespace string) (int, error) {
+	logrus.Infof("Getting the IO Speed in pod %s", podName)
+	pod, err := k8sCore.GetPodByName(podName, namespace)
+	if err != nil {
+		return 0, fmt.Errorf("error in getting FIO PODS")
+	}
+	logOptions := corev1.PodLogOptions{
+		TailLines: getInt64Address(15),
+	}
+	log, err := k8sCore.GetPodLog(pod.Name, pod.Namespace, &logOptions)
```

Comment:
Echo Vilas comments below, to better solve it and save a lot of parsing here. It's best to have fio output to be json format and then you can simply extract the values. e.g.
fio --blocksize=4k --filename=/dev/pxd/pxd713287577393395774 --ioengine=libaio --readwrite=read --size=256MB --name=test --direct=1 --gtod_reduce=1 --iodepth=128 --allrandrepeat=1 --end_fsync=1 --disable_lat=0 --gtod_reduce=0 --output-format json --output=fio_out 1>/dev/null && cat fio_out | jq -r '.jobs[0] | .read.iops,.read.bw_bytes,.write.iops,.write.bw_bytes'

Reply:
You can also use `--output-format=json` parameter in fio

Reply:
watch out the output might be different from different fio version. suggest check 
"fio version" : "fio-2.2.8",
as well as 
 "fio version" : "fio-3.7",
 so that you save one trip to fix the test later when the fio image was updated.